src/server/socketHandlers/scConnectionHandler.js

```diff
@@ -2,23 +2,49 @@ import scSubscribeHandler from './scSubscribeHandler';
 import scUnsubscribeHandler from './scUnsubscribeHandler';
 import scGraphQLHandler from './scGraphQLHandler';
 import {REFRESH_JWT_AFTER} from 'server/utils/serverConstants';
+import getRetink from 'server/database/rethinkDriver';
+import isObject from 'universal/utils/isObject';
+import jwtDecode from 'jwt-decode';
 
+// we do this otherwise we'd have to blacklist every token that ever got replaced & query that table for each query
+const istmsValid = (tmsFromDB, tmsFromToken) => {
+  if (tmsFromDB.length !== tmsFromToken.length) return false;
+  for (let i = 0; i < tmsFromDB.length; i++) {
+    if (tmsFromDB[i] !== tmsFromToken[i]) return false;
```

Comment:
Are we sure that the order of these arrays will always match? We don't need to add a sort here?